### PR TITLE
planner event publishing

### PR DESCRIPTION
**Pull request by elegantmoose**
- changes to event_svc:
    - make it similar to RabbitMQ API/terminology w/ usage of 'exchange' and 'queue' abstractions.
    - timestamp is added to every event
- planning service publishes bucket transitions by default, if `execute_planner()` method is used to run the planner. (There isnt another way to globally check when a bucket transition occurs while executing the planner other than this method as it actually executes the next bucket that the planner wants)
- Also, just some clean up of planning service. (removed method and changed a method name as the name implied different action being taken than actually was)

### CODE REVIEW

**Comment by christophert** [comment]
@elegantmoose please fix build errors - looks to be just some styling issues

**Comment by elegantmoose** [comment]
you beat me before I could push lol

**Review by christophert — APPROVED** [review]

File: app/service/planning_svc.py [hunk 1]
```diff
@@ -112,11 +115,26 @@ async def execute_planner(self, planner):
...
+          events to the event service
+        :type publish tranitions: bool
         """
+        async def _publish_bucket_transition(bucket):
+            """ subroutine to publish bucket transitions to event_svc"""
+            await self.publish_event(event='bucket',
+                                     msg=f'Bucket transition: {bucket}',
+                                     operation_id=planner.operation.id,
+                                     operation_name=planner.operation.name,
+                                     ts=str(datetime.datetime.now()))
```
Comment (wbooth) [inline app/service/planning_svc.py:128]: event_time, may be a useful global

**Comment by elegantmoose** [comment]
Started requested changes.

Also, note that the cleaning done on planning service here, will break new tests in https://github.com/mitre/caldera/pull/1896. Thus if that PR gets merged first, then the broken tests need to be fixed here.

**Review by wbooth — DISMISSED** [review]
pending discussed changes

**Comment by elegantmoose** [comment]
@wbooth - merge https://github.com/mitre/caldera/pull/1896/files first, then Ill pull those changes into this branch, fix tests, then should be good.

**Comment by elegantmoose** [comment]
@wbooth - set.

**Pull request merged**